Makefile

```
@@ -709,3 +709,32 @@ $(CODESPELL): $(PYTHON_VENV_STAMP)
+.PHONY: test-ui-headed
+test-ui-headed: test-ui-install ## Run UI tests in headed mode (interactive)
+	@echo "Running UI tests in headed mode..."
+	cd test/ui && npm run test:headed
+
+.PHONY: test-ui-common
+test-ui-common: test-ui-install ## Run common UI tests (smoke tests, infrastructure validation)
+	@echo "Running common UI tests..."
+	cd test/ui && npm run test:common
+
+.PHONY: test-ui-agent
+test-ui-agent: test-ui-install ## Run Agent platform UI tests
```

checkmake (0.3.2)

[warning] Required target "all" must be declared PHONY.

(minphony)